### PR TITLE
VOOD-150: Capture ctrl+click event on variable node.

### PR DESCRIPTION
**Pull request by GinoCardillo-OST**

### CODE REVIEW

File: media/html/visjsDebuggerPanel.html [hunk 1]
```diff
@@ -66,6 +66,15 @@
         network = new vis.Network(container, data, message.options);
         vscode = acquireVsCodeApi();
         canvas = document.querySelector('#visjs-network canvas');
+
+        network.on('click', function (properties) {
+          const isCtrlClicked = properties.event.srcEvent.ctrlKey;
+          const ids = properties.nodes;
+          if (isCtrlClicked && ids?.length === 1 && vscode) {
```
Comment (alexandre-lagadec-ost) [inline media/html/visjsDebuggerPanel.html:73]:
```suggestion
          if (isCtrlClicked && ids?.length === 1) {
```
`vscode` is always defined at this point (see line 67).

File: src/debug-adapter/debugEventManager.ts [hunk 2]
```diff
@@ -98,7 +102,7 @@ export class DebugEventManager {
     panelViewInput: PanelViewInput
   ): Promise<void> {
     let isNewAndObject = false;
-    const id = variable.value;
+    const id = variable.value === 'null' ? variable.value : `${DebugEventManager.objectPrefix}${variable.value}`;
```
Comment (alexandre-lagadec-ost) [inline src/debug-adapter/debugEventManager.ts:105]: Is the prefix (none or `DebugEventManager.objectPrefix`) always consistent with `isNewAndObject` (1/2)?

File: src/debug-adapter/debugEventManager.ts [hunk 3]
```diff
@@ -135,10 +139,11 @@ export class DebugEventManager {
     variable: DebugProtocol.Variable
   ): Promise<[variable: PanelViewVariable, isNewAndObject: boolean]> {
     let isNewAndObject = false;
-    const panelViewVariable: PanelViewVariable = { id: id !== 'null' ? id : hash(variable) };
+    const panelViewVariable: PanelViewVariable = { id: id !== 'null' ? id : `${DebugEventManager.variablePrefix}${hash(variable)}` };
```
Comment (alexandre-lagadec-ost) [inline src/debug-adapter/debugEventManager.ts:142]: Is the prefix (none or `DebugEventManager.objectPrefix`) always consistent with `isNewAndObject` (2/2)?

File: src/debug-adapter/debugEventManager.ts (hunk 2 above)
Comment (GinoCardillo-OST) [reply src/debug-adapter/debugEventManager.ts:105]: No, It can be an existing object

**Pull request closed without merging**